src/utils/shutdown_token.rs

```diff
@@ -12,15 +12,15 @@ use tokio::sync::Notify;
 
 use crate::dispatching::update_listeners::UpdateListener;
 
-/// A token which used to shutdown [`Dispatcher`].
+/// A token which used to shutdown [`crate::dispatching::Dispatcher`].
```

Comment:
```suggestion
/// A token which used to shutdown [`Dispatcher`].
///
/// [`Dispatcher`]: crate::dispatching::Dispatcher
```

Reply:
I experienced some doc warnings when running it locally. With a full path, it works smoothly.

Reply:
With my proposed change it's still the full path, it's just isn't shown in the documentation.